hadoop/src/main/java/org/apache/carbondata/hadoop/api/CarbonFileInputFormat.java

```diff
@@ -208,6 +214,18 @@ public CarbonTable getOrCreateCarbonTable(Configuration configuration) throws IO
     return carbonFiles;
   }
 
+  private List<CarbonFile> getAllCarbonDataFiles(List fileLists) {
+    List<CarbonFile> carbonFiles = new LinkedList<>();
+    try {
```

Comment:
This change is not related to binary, suggest to keep it in separate PR

Reply:
ok, done